docs/core/compatibility/aspnet-core/6.0/razor-engine-apis-obsolete.md

```diff
@@ -31,9 +31,9 @@ Migrate source code from the `RazorEngine` type to the `RazorProjectEngine` type
 
 ## Affected APIs
 
-- [Microsoft.AspNetCore.Mvc.Razor.Extensions.InjectDirective.Register](/dotnet/api/microsoft.aspnetcore.mvc.razor.extensions.injectdirective.register?view=aspnetcore-3.1&preserve-view=true)
-- [Microsoft.AspNetCore.Mvc.Razor.Extensions.ModelDirective.Register](/dotnet/api/microsoft.aspnetcore.mvc.razor.extensions.namespacedirective.register?view=aspnetcore-2.2&preserve-view=true)
-- [Microsoft.AspNetCore.Mvc.Razor.Extensions.PageDirective.Register](/dotnet/api/microsoft.aspnetcore.mvc.razor.extensions.namespacedirective.register?view=aspnetcore-2.2&preserve-view=true)
```

Comment:
Why they are not available anymore in the API reference?

Reply:
They were removed in [this PR](https://github.com/dotnet/AspNetApiDocs/pull/206). @v-alje Was this intentional?

Reply:
@gewarren These are in the Microsoft.AspNetCore.Mvc.Razor.Extensions NuGet package which was requested to be removed with the note (only useful for code-gen) in the thread. Do we need to add Microsoft.AspNetCore.Mvc.Razor.Extensions back in to either the dotnet or ASP repos?
+ @JamesNK

Reply:
I double checked with @pranavkm and the docs are ok to be filtered out. These APIs shouldn't be used by developers.

The change in this PR is fine.

Reply:
@JamesNK Thanks for checking!